src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Authentication.cs

```diff
@@ -90,6 +90,31 @@ public abstract class HttpClientHandler_Authentication_Test : HttpClientTestBase
             }, options);
         }
 
+        [Theory]
+        [InlineData("WWW-Authenticate: Basic realm=\"hello\"\r\nWWW-Authenticate: Negotiate\r\nx-identifier: Test\r\n", "Basic")]
+        [InlineData("WWW-Authenticate: Basic realm=\"hello\"\r\nWWW-Authenticate: Digest realm=\"hello\", nonce=\"hello\", algorithm=MD5\r\nWWW-Authenticate: Negotiate\r\nx-identifier: Test\r\n", "Digest")]
+        public async Task HttpClientHandler_MultipleAuthenticateHeaders_PicksSupported(string authenticateHeader, string supportedAuth)
+        {
+            if (IsCurlHandler && authenticateHeader.Contains("Digest"))
+            {
+                // TODO: #27113: Fix failing authentication test cases on different httpclienthandlers.
```

Comment:
Change issue number to #28065. #27113 is closed.

Reply:
Will do!